image/scanner/rhel/konflux.Dockerfile

```
@@ -7,7 +7,7 @@ ARG BASE_TAG=latest
 FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_1.21 AS builder
 
 ARG SCANNER_TAG
-ENV RELEASE_TAG="${SCANNER_TAG}"
+ENV RELEASE_TAG="${SCANNER_TAG}-${TARGETARCH}"
```

I'd suggest, let's keep `RELEASE_TAG` as it was. There's some elaborate handling where its value gets embeeded in Go binary and later used in some code (the fanout is a bit wide and deep so I procrastinate going there).
```suggestion
ENV RELEASE_TAG="${SCANNER_TAG}"
```